src/components/eventsSection/ActionContainer.tsx

```diff
@@ -0,0 +1,49 @@
+import Link from 'next/link'
+import Image from 'next/image'
+import { EventsSectionType } from '~/lib/sanity.queries/eventsSection/types'
+import SecondaryButton from '../SecondaryButton'
+
+interface ActionSectionProps {
+  section: EventsSectionType
+}
+
+export default function ActionContainer({ section }: ActionSectionProps) {
+  const backgroundImageUrl =
+    typeof section.ctaBackgroundImage === 'string'
```

Comment:
it is very strange,
it should work as everywhere else ` src={urlForImage(backgroundImage)?.url() || ''}`